requirements-dev.txt

```diff
@@ -8,3 +8,4 @@ pytest-cov==2.7.1
 pytest-mock==1.10.4
 pytest==4.6.2
 requests-mock==1.6.0
+pdbpp
```

Comment:
was pdbpp added just temporarily or do we want it for good? If we want it for good, don't we need to pin version?

Reply:
i'll pin version. it's a useful improvement over pdb